src/MainView.mc

```diff
@@ -98,11 +104,27 @@ class MainView extends WatchUi.View {
     }
   }
 
+  function drawProgress(dc, value, max, codeColor) {
+    dc.setPenWidth(dc.getHeight() / 40);
+    dc.setColor(codeColor, Graphics.COLOR_TRANSPARENT);
+    if (screen_shape_== System.SCREEN_SHAPE_ROUND) {
+      if ( _enableAntiAlias ) {
+        dc.setAntiAlias(true);
+      }
+      dc.drawArc(dc.getWidth() / 2, dc.getHeight() / 2, (dc.getWidth() / 2) - 2, Graphics.ARC_COUNTER_CLOCKWISE, 90, ((value * 360) / max) + 90);
+      if ( _enableAntiAlias ) {
+        dc.setAntiAlias(false);
+      }
+    } else if (screen_shape_ == System.SCREEN_SHAPE_RECTANGLE) {
```

Comment:
Why not render this progress bar in all other screen shapes? i.e. drop the `if` .. afaik there is at least the semiround screen shape as well.